### ISSUE TITLE
Configurability n such

### ISSUE THREAD

**Issue by robwormald** [balderdashy/sails.io.js#2]
The other day we talked about using this module as the common-core for all client-side sdk type things. I'm all for this w/ angularSails, and had a go at it this morning.

The issue at the moment is that I can't really find a good way to get into the guts of sails.io.js - since its sort of all enclosed by the IIFE's. 

Use cases:
- adding auth type params to the initial connection
- connecting the socket at some point after launch.
- instantiating multiple connections to same/different io servers.

I might be missing something - suggestions? At the moment I'm happy to track changes to this repo manually, but as you said it would def be a nice to have.

Alternately I could port the basic angular wrapper stuff into the this repo, and use grunt to build out a couple of different versions.

R

**Comment by sailsbot** [comment]
Thanks for posting, @robwormald.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
